dashboard/src/components/AppList/AppList.v2.tsx

```diff
@@ -50,51 +50,54 @@ function AppList(props: IAppListProps) {
   useEffect(() => {
     fetchAppsWithUpdateInfo(namespace, true);
     getCustomResources(namespace);
+  }, [namespace, fetchAppsWithUpdateInfo, getCustomResources]);
+
+  useEffect(() => {
     setFilter(filterProps);
-  }, [namespace, filterProps, fetchAppsWithUpdateInfo, getCustomResources]);
+  }, [filterProps]);
 
   return (
     <section>
       <PageHeader>
-        <Column span={10}>
+        <div className="kubeapps-applist-header">
           <Row>
-            <>
-              <h1>Applications</h1>
-              <SearchFilter
-                key="searchFilter"
-                placeholder="search apps..."
-                onChange={setFilter}
-                value={filter}
-                onSubmit={pushSearchFilter}
-              />
-            </>
+            <Column span={10}>
+              <Row>
+                <h1>Applications</h1>
+                <SearchFilter
+                  key="searchFilter"
+                  placeholder="search apps..."
+                  onChange={setFilter}
+                  value={filter}
+                  onSubmit={pushSearchFilter}
+                />
+              </Row>
```

Comment:
I think this `Row` is not required

Reply:
It is to avoid having the `h1` and the filter in different lines.